rocketmq-iot-bridge/src/main/java/org/apache/rocketmq/iot/MQTTBridge.java

```diff
@@ -101,20 +124,22 @@ private void registerMessageHandlers() {
 
     public void start() {
         try {
+            publishProducer.start();
             ChannelFuture channelFuture = serverBootstrap.bind().sync();
             channelFuture.channel().closeFuture().sync();
+            logger.info("start the MQTTServer success.");
```

Comment:
It would be better print all necesarry configuration when you starting server. Only success or ok is too little significant.

Reply:
I'll fix it~